lib/templates/error.html

```diff
@@ -23,14 +23,22 @@
         padding: 0;
       }
       .ember-cli-logo {
+        margin-top: 50px;
         width: 250px;
+        float: left;
+      }
+      .tomster-logo {
+        height: 200px;
+        overflow: hidden;
+        float: right;
       }
       .content {
         padding: 0 15px;
       }
       .container {
         width: 1080px;
         margin: 0 auto;
+        overflow: hidden;
```

Comment:
can you explain why you added this? 🤔

Reply:
I wanted to make sure it wouldn't two-line the logos, i.e. move Tomster up and under the ember-cli logo inadvertently.

Reply:
Let me know if you want some CSS adjustments

Reply:
yeah, I'm thinking about it 😉 

once I find time to actually try this change out I'll let you know :)

Reply:
No hurry.

Reply:
Oh, hey. I tried removing the `overflow: hidden;`, which popped "DANG" up in between the ember-CLI logo and Tomster with certain screen sizes (so I put it back). There are certainly other css formatting options, but this is the one I went with. That said, now that Tomster has feet again, let me know if you want a css change.